src/components/ErrorNotification/ErrorNotification.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { useState } from 'react';
+import { TextError } from '../../types/TextError';
+
+type Props = {
+  error: TextError;
+  setError: (errorMessage: TextError | null) => void;
+};
+
+export const ErrorNotification: React.FC<Props> = ({ error, setError }) => {
+  const [errorText] = useState(TextError.Add);
+
+  const closeError = () => {
+    setError(null);
+  };
+
+  return (
+    <div
+      data-cy="ErrorNotification"
+      className="notification is-danger is-light has-text-weight-normal"
+    >
+      <button
+        data-cy="HideErrorButton"
+        type="button"
+        className="delete"
```

Comment:
Where is `hidded` class?
<img width="881" alt="Screenshot 2022-10-10 at 21 21 41" src="https://user-images.githubusercontent.com/17098736/194929912-b8860593-6201-4afb-a158-b811bd328a97.png">

Be always attentive with tasks descriptions and checklists. There **no redundant** words. The same would be in test tasks. You need to read it attentively and write the code according to it. It is important to pass all employment stages and get the job 😉